cmd/xdpcap/flags.go

```diff
@@ -4,11 +4,53 @@ import (
 	"flag"
 	"fmt"
 	"os"
+	"strconv"
 	"strings"
 
 	"github.com/pkg/errors"
 )
 
+type actionsFlag []xdpAction
+
+func (a *actionsFlag) Set(val string) error {
+	// Clear defaults
+	*a = (*a)[:0]
+
+	for _, action := range strings.Split(val, ",") {
+		action = strings.TrimSpace(strings.ToLower(action))
+
+		switch action {
+		case "aborted":
+			*a = append(*a, xdpAborted)
+		case "drop":
+			*a = append(*a, xdpDrop)
+		case "pass":
+			*a = append(*a, xdpPass)
+		case "tx":
+			*a = append(*a, xdpTx)
+		default:
+			// Accept actions we don't know about using their numeric value
+			unknownAction, err := strconv.Atoi(action)
+			if err != nil {
+				return errors.Errorf("unknown action %s", action)
+			}
+
+			*a = append(*a, xdpAction(unknownAction))
+		}
+	}
+
+	return nil
+}
+
+func (a *actionsFlag) String() string {
+	strs := []string{}
+	for _, action := range *a {
+		strs = append(strs, action.String())
```

Comment:
This isn't going to produce the input, but something like `XDPPass,XDPDrop` no?

Reply:
I used stringer with `-stripprefix`, so it did print `Pass`, `Drop`.. . This wasn't very clear though, and `stringer` doesn't support parsing stuff, so I've gotten rid of string and manually created the `xdpAction.String()` and `parseAction()` functions.